check.go

```diff
@@ -13,24 +13,33 @@ type CheckOwner struct {
 }
 
 type Check struct {
-	Category    Category   `graphql:"category"`
-	Description string     `graphql:"description"`
-	Enabled     bool       `graphql:"enabled"`
-	Filter      Filter     `graphql:"filter"`
-	Id          graphql.ID `graphql:"id"`
-	Level       Level      `graphql:"level"`
-	Name        string     `graphql:"name"`
-	Notes       string     `graphql:"notes"`
-	Owner       CheckOwner `graphql:"owner"`
-	Type        CheckType  `graphql:"type"`
-
-	CustomEventCheckFragment      `graphql:"... on CustomEventCheck"`
-	ManualCheckFragment           `graphql:"... on ManualCheck"`
-	RepositoryFileCheckFragment   `graphql:"... on RepositoryFileCheck"`
-	RepositorySearchCheckFragment `graphql:"... on RepositorySearchCheck"`
-	ServicePropertyCheckFragment  `graphql:"... on ServicePropertyCheck"`
-	TagDefinedCheckFragment       `graphql:"... on TagDefinedCheck"`
-	ToolUsageCheckFragment        `graphql:"... on ToolUsageCheck"`
+	Category    Category     `graphql:"category"`
```

Comment:
Is there a reason to not include campaign here?

Reply:
We are not sure we are building support into the tools for that yet.

Also if we did we'd do it as a separate initiative